### PR TITLE
Example for rasterize use.

### PR DESCRIPTION
**Pull request by awage**
# Description

This is just a small contribution to the docs for the use of the rasterize function. I might be helpfull for other users.

### CODE REVIEW

File: docs/documentation/backends/cairomakie.md [hunk 1]
```diff
@@ -55,3 +55,10 @@ Assuming that you have a `Plot` object `plt`, you can set `plt.rasterize = true`
 For example, if your Scene's resolution is `(800, 600)`, by setting `scale=2`, the rasterized image will have a resolution of `(1600, 1200)`.
 
 You can deactivate this rasterization by setting `plt.rasterize = false`.
+
+Example: 
+``` 
+fig = Figure()
+scatter(fig[1,1], v[:,1], v[:,2], rasterize = true, markersize = 1.)
+save("raster_test.pdf", fig)
+```
```
Comment (asinghvi17) [inline docs/documentation/backends/cairomakie.md:64]:
```suggestion
\begin{examplefigure}{format=svg}
fig = Figure()
scatter(fig[1,1], v[:,1], v[:,2], rasterize = true, markersize = 1.)
\end{examplefigure}

Comment (asinghvi17) [reply docs/documentation/backends/cairomakie.md:64]: This is only if you want the figure to show up in the docs.

File: docs/documentation/backends/cairomakie.md [hunk 2]
```diff
@@ -55,3 +55,10 @@ Assuming that you have a `Plot` object `plt`, you can set `plt.rasterize = true`
 For example, if your Scene's resolution is `(800, 600)`, by setting `scale=2`, the rasterized image will have a resolution of `(1600, 1200)`.
 
 You can deactivate this rasterization by setting `plt.rasterize = false`.
+
+Example: 
+\begin{examplefigure}{format=svg}
+fig = Figure()
+v = rand(10000,2)
+scatter(fig[1,1], v[:,1], v[:,2], rasterize = true, markersize = 1.0)
```
Comment (asinghvi17) [inline docs/documentation/backends/cairomakie.md:63]:
```suggestion
v = rand(10000,2)
scatter(v[:,1], v[:,2], rasterize = true, markersize = 1.0)
```

It looks like `scatter` does not return a save-able value, so hopefully this helps.

**Comment by asinghvi17** [comment]
Will merge once the latest commit builds.  Thanks @awage!

**Comment by awage** [comment]
Thank you for the feature! it is very helpful!

**Review by asinghvi17 — APPROVED** [review]

**Pull request merged**